libdnf/module/ModulePackageContainer.hpp

```diff
@@ -172,6 +172,15 @@ struct ModulePackageContainer
 
     bool isDisabled(const std::string &name);
     bool isDisabled(const ModulePackage * module);
+
+    /**
+     * @brief Report if a platform module was added to ModulePackageContainer. Return true if the
+     * platform module was added
+     *
+     * @return bool
+     */
```

Comment:
```
   /**
     * @brief Report whether a platform module was added to ModulePackageContainer.
     *
     * @return true if the platform module was added
     */
```

I'd change the docstring like this. Using "whether" instead of "if" here is much clearer.